add missing link to QScintilla library to correctly build Python bindings

author: Kenneth Hoste (HPC-UGent)
author: Pavel Grochal (INUITS)
--- QScintilla_gpl-2.11.2/Python/configure.py.orig	2019-10-03 17:28:55.552134970 +0200
+++ QScintilla_gpl-2.11.2/Python/configure.py	2019-10-03 17:29:54.942850767 +0200
@@ -1612,7 +1612,7 @@
 
     libs = qmake_config.get('LIBS')
     if libs:
-        pro.write('LIBS += %s\n' % libs)
+        pro.write('LIBS += %s -lqscintilla2_qt5\n' % libs)
 
     if not opts.static:
         dylib = module_config.get_mac_wrapped_library_file(target_config)
